controller/clusters_blackbox_test.go

```diff
@@ -23,45 +19,13 @@ func TestRunClustersREST(t *testing.T) {
 	suite.Run(t, &TestClustersREST{UnitTestSuite: testsuite.NewUnitTestSuite()})
 }
 
-func (rest *TestClustersREST) SecuredControllerWithServiceAccount(serviceAccount account.Identity) (*goa.Service, *ClustersController) {
-	svc := testsupport.ServiceAsServiceAccountUser("Token-Service", serviceAccount)
+func (rest *TestClustersREST) UnsecuredController() (*goa.Service, *ClustersController) {
```

Comment:
why are you changing the controller to `unsecured` if you need a SA token in https://github.com/fabric8-services/fabric8-auth/pull/659/files#diff-28264df4d2b218e146d7d2540ac6c26dR92 ? (unless I missed something?)